package.json

```diff
@@ -5,22 +5,10 @@
   "dependencies": {
     "@react-firebase/database": "^0.3.11",
     "@react-google-maps/api": "^2.2.0",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/google-map-react": "^2.1.0",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^16.0.0",
-    "@types/react": "^16.9.53",
-    "@types/react-alert": "^5.2.0",
-    "@types/react-dom": "^16.9.8",
-    "@types/styled-components": "^5.1.7",
     "firebase": "^8.2.4",
     "firestore-export-import": "^0.9.0",
     "google-map-react": "^2.1.9",
-    "install": "^0.13.0",
```

Comment:
not sure what "install" was used for? assuming you deleted it on purpose!

Reply:
I couldn't find it being used anywhere, so I removed it. I'm not sure what it was supposed to be for, either!